ghost/core/core/server/data/migrations/versions/5.19/2022-10-10-10-04-add-members-feedback-table.js

```diff
@@ -0,0 +1,10 @@
+const {addTable} = require('../../utils');
+
+module.exports = addTable('members_feedback', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    type: {type: 'string', maxlength: 50, nullable: false, defaultTo: 'positive', validations: {isIn: [['positive', 'negative']]}},
```

Comment:
Just FYI the `validations` config is not used at the DB level but at the model level, so it's not necessary here.